apps/trading/lib/hooks/use-vega-transaction-toasts.tsx

```diff
@@ -474,15 +474,17 @@ const VegaTxCompleteToastsContent = ({ tx }: VegaTxToastContentProps) => {
   }
 
   if (tx.order && tx.order.rejectionReason) {
+    const rejectionReason = getRejectionReason(tx.order) || ' ';
     return (
       <>
         <ToastHeading>{t('Order rejected')}</ToastHeading>
-        <p>
-          {t(
-            'Your order has been rejected because: %s',
-            getRejectionReason(tx.order) || ''
-          )}
-        </p>
+        {rejectionReason ? (
```

Comment:
Fallback. But we need this ticket merged https://github.com/vegaprotocol/vega/pull/8046 @mattrussell36  and @candida-d

Reply:
And after that generate again types on our end.